docs/configuring-ingest.asciidoc

```diff
@@ -0,0 +1,87 @@
+// This file was copied over from libbeat and
+// then adapted to APM Server specific needs
+
+[[configuring-ingest-node]]
+== Parse data using ingest node pipelines
+
+When you use Elasticsearch for output, you can configure {beatname_uc} to use
```

Comment:
Alternative suggestion:

With the Elasticsearch output you can configure the APM Server to use
{elasticsearch}/ingest.html[ingest node] to pre-process documents before indexing them. 
A pipeline defines processors that operate on your data. 
For example, you can register a pipeline that defines one processor that removes a field and another that renames other field.